rosplane/launch/rosplane.launch.py

```diff
@@ -23,20 +20,15 @@ def generate_launch_description():
         if arg.startswith("aircraft:="):
             aircraft = arg.split(":=")[1]
         
-        if arg.startswith("init_lat:="):
-            init_lat = float(arg.split(":=")[1])
-            assert isinstance(init_lat, float)
-            init_lat = str(init_lat)
+        if arg.startswith("seed_estimator:="):
+            use_params = arg.split(":=")[1].lower()
 
-        if arg.startswith("init_long:="):
-            init_long = float(arg.split(":=")[1])
-            assert isinstance(init_long, float)
-            init_long = str(init_long)
+            print(use_params)
```

Comment:
Could we change this print statement to have more meaningful output, like 
"Seeding estimator with saved params: {use_params}"?

Otherwise, there's just a couple of bool values that get printed to the screen.

Reply:
This is a little nitpicky, but it currently prints out twice. After some testing, it looks like the `generate_launch_description` is being called twice when launched. I don't think this is anything we are doing, but rather the launching sequence that is responsible for taking the launch file and doing something with the `LaunchDescription` (which I don't know anything about).

However, we may want to move these warnings into the constructor of the estimator instead of the launch file? Otherwise we'll have the same information printed to the screen twice. Not a huge deal, but would be cleaner if we printed in a different place. Thoughts?

![image](https://github.com/rosflight/rosplane/assets/56745254/01a94e50-097e-4993-b62f-7d2cd98a719f)

Reply:
I think the error checking should definitely be moved inside the C++ file. I will do that.

Reply:
I think I know why it is printing twice, when you use `sim.launch.py` it calls `rosplane.launch.py` which calls `generate_launch_description` and then as `rosplane.launch.py` returns, `sim.launch.py` calls `generate_launch_description` again. That is my theory anyway. Its purely academic because the error checking should be in the C++ file anyway.

Reply:
Moved it!